.github/scripts/platform_lib.py

```diff
@@ -168,6 +185,15 @@ def find_all_ci_instances(self) -> List:
         all_ci_instances = get_instances_with_filter([all_ci_instances_filter], allowed_states=['*'])
         return all_ci_instances
 
+    def find_run_farm_ci_instances(self, workflow_tag: str = '*') -> List:
+        # on AWS run farm instances are marked with 'fsimcluster'
+        instances_filter = [
+                {'Name': 'tag:fsimcluster', 'Values': f'*{workflow_tag}*'},
```

Comment:
Since we put the `GITHUB_RUN_ID` into the cluster name, we can then use this tag to see what FPGA instances are running.